package.json

```diff
@@ -8,7 +8,7 @@
     "flow": "flow",
     "lint": "eslint --ext .js --ext .jsx .",
     "build-dev": "webpack --config webpack.dev.js",
-    "build-prod": "npm-run-all clean validate test 'webpack --config webpack.prod.js --env.prod'",
```

Comment:
Why have these been removed?

Reply:
Thanks for spotting! This was for speeding up development, as you have to run `yarn build-prod` to get Apple Pay to work locally